app/views/maps/index.html.erb

```diff
@@ -42,7 +42,7 @@
 
       <div>
         <br />
-        <%= render :partial => "maps/search" %>
+        <%= render :partial => "layouts/header" %>
```

Comment:
Oh cool! Will this put a header in here instead of the search partial? Can you share a screenshot? thank you!!!

Reply:
Hi @jywarren, this was done because the search and feature methods in the maps_controller both point to maps/index.  The index.html.erb pictured above was looking to render the maps/search which was where the file that I deleted was (the second search bar).  We needed it to point to the correct search bar which is in the layouts views in the header file so that all tests would pass.  I believe the navbar Search Bar was only acting the same because it shared the same id and name as the second search bar.  It wasn't actually set up to pass the tests.  The second one was.  Should some renaming be done somewhere in order to have it make more sense?

Reply:
I tested it and everything works the same as before but now with the second search bar removed.  It looks much cleaner.
![image](https://user-images.githubusercontent.com/29240939/57940939-91081200-789b-11e9-8a2c-a035452273c4.png)